### PR TITLE
Update bundle_de.properties to latest keys in bundle

### PR DESCRIPTION
**Pull request by hype11**
add new keys

### CODE REVIEW

File: plugins/org.jkiss.dbeaver.data.gis.view/OSGI-INF/l10n/bundle_de.properties [hunk 1]
```diff
@@ -1,10 +1,13 @@
...
 
 extension-point.org.jkiss.dbeaver.ui.gis.panel.GISTextViewer.label = Text
 extension-point.org.jkiss.dbeaver.ui.gis.panel.GISTextViewer.description = Zeigen Sie das GIS-Objekt in Textform an
 extension-point.org.jkiss.dbeaver.ui.gis.panel.GISBrowserViewer.label = Karte
-extension-point.org.jkiss.dbeaver.ui.gis.panel.GISBrowserViewer.description = GIS-Objekt auf der Karte anzeigen
\ No newline at end of file
+extension-point.org.jkiss.dbeaver.ui.gis.panel.GISBrowserViewer.description = GIS-Objekt auf der Karte anzeigen
+
+pref.page.name.gis = GIS Betrachter
+keyword.org.jkiss.dbeaver.preferences.gis = GIS SRID CRS Spatial
```
Comment (LonwoLonwo) [inline plugins/org.jkiss.dbeaver.data.gis.view/OSGI-INF/l10n/bundle_de.properties]: Could you please also add a translation of the word "Spatial" here?

Comment (hype11) [reply plugins/org.jkiss.dbeaver.data.gis.view/OSGI-INF/l10n/bundle_de.properties]: Spatial is a special word that is also in germany widely used to describe geo data. But I tried a translation

**Review by LonwoLonwo — APPROVED** [review]

**Review by E1izabeth — APPROVED** [review]

**Comment by LonwoLonwo** [comment]
For testers: This can be checked on the GIS Viewer preference page name.

**Comment by Matvey16** [comment]
Verified

**Pull request merged**